hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/AttributeSearchIntegrationTest.kt

```diff
@@ -54,7 +54,7 @@ class AttributeSearchIntegrationTest : AbstractSearchIntegrationTest() {
     ),
     PrisonerBuilder(
       prisonerNumber = "P2",
-      firstName = "John2",
+      firstName = "James2",
```

Comment:
Had to make the names more different as they were being picked up by the fuzzy searches.